thingsboard_gateway/connectors/ble/device.py

```diff
@@ -151,7 +164,7 @@ async def __process_self(self):
                         self.__set_char_handle(item, char_id)
                         self.notifying_chars.append(char_id)
                         await self.notify(char_id)
-                    except BleakError:
+                    except BleakError as e:
                         log.error(e)
```

Comment:
What if we receive an error during device reading? Could we schedule a reconnect and reading/ subscribing for notification after receiving an error?